readme.md

```diff
@@ -30,8 +30,8 @@ Make all the changes smooth on hover (during 300ms):
 ## Checklist
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
-- [DEMO LINK](https://<your_account>.github.io/layout_catalog/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_catalog/report/html_report/)
+- [DEMO LINK](https://wintsistrs.github.io/layout_catalog/)
+- [TEST REPORT LINK](https://wintsistrs.github.io/layout_catalog/report/html_report/)
```

Comment:
Add these links to the PR description 
<img width="936" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/64750958/d1c04888-f0bd-4b9e-9333-e35cc7ca1bd6">